### PR TITLE
Add ManualSpeedTestITCase For Assessing State Performance

### PR DESCRIPTION
**Pull request by aljoscha**
This should be used to test whether there are any obvious performance
regressions between releases. Somewhat similar to the other manual tests
that have @Ignore set these have to be run manually. (for now)

R: @rmetzger for review

### CODE REVIEW

File: flink-tests/src/test/java/org/apache/flink/test/state/ManualWindowSpeedITCase.java [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+ * been processed in each window. This gives an estimate of the throughput.
+ */
+@Ignore
+public class ManualWindowSpeedITCase extends StreamingMultipleProgramsTestBase {
+
+	@Rule
+	public TemporaryFolder tempFolder = new TemporaryFolder();
+
+	@Test
+	public void testTumblingIngestionTimeWindowsWithFsBackend() throws Exception {
```
Comment (rmetzger) [inline flink-tests/src/test/java/org/apache/flink/test/state/ManualWindowSpeedITCase.java]: Is the FsBackend the default one now?

Comment (aljoscha) [inline flink-tests/src/test/java/org/apache/flink/test/state/ManualWindowSpeedITCase.java]: Ah no, it's not, but they use the same mechanism for keyed state, that's why I didn't set it explicitly. Will change.

**Comment by aljoscha** [comment]
@rmetzger I changed to `FsStateBackend`. What do you think now?

**Comment by rmetzger** [comment]
Merge it :)

**Pull request closed without merging**